.drone-v1.yml

```diff
@@ -8,14 +8,40 @@ platform:
   arch: amd64
 
 steps:
+  - name: build
+    pull: if-not-exists
+    image: quay.io/ukhomeofficedigital/scala-sbt-nodejs:v1.0.0
+    commands:
+      - /root/entrypoint.sh
+      - sbt --error compile docker:stage
+      - sbt -ivy ./.ivy2 dependencyCheck
```

Comment:
We might want to consider moving these additional commands to a new step that can be run in parallel with the `unit-test` and `publish-drt` so the builds don't slow down - they're currently about 4 minutes longer because of these